### PR TITLE
Updates to U2F API & WebAuthn

### LINKED ISSUE #6217: U2F API deprecated in Chrome v. 98
Please update the website a like.

Would be good to add distinction between the standards by adding FIDO2/WebAuthm API to the website separately. Currently only U2F API is mentioned which has stopped working.

https://www.yubico.com/blog/google-chrome-u2f-api-decommission-what-the-change-means-for-your-users-and-how-to-prepare/

### PR DESCRIPTION
**Pull request by Schweinepriester**
Fixes #6217.

Notes:

- links I haven't added which might be helpful to some:
  - https://developer.chrome.com/blog/deps-rems-95/#deprecate-u2f-api-cryptotoken
  - https://chromeenterprise.google/intl/en/policies/#U2fSecurityKeyApiEnabled
  - https://developer.chrome.com/origintrials/#/view_trial/-6366963973195038719
- I went with Chrome 106 because
  - they were talking about it here: https://groups.google.com/a/chromium.org/g/blink-dev/c/xHC3AtU_65A/m/27jV8fVsAgAJ
  - didn't seem they finalised the discussion
  - but https://developer.chrome.com/origintrials/#/view_trial/-6366963973195038719 includes 105, so I guess it was accepted
- potential todo: align Edge & Opera
  - Edge seems supportive here: https://groups.google.com/a/chromium.org/g/blink-dev/c/xHC3AtU_65A/m/Ai-evvu5AgAJ
  - but no other sign, so as opposed to chrome even less indication and since we don't seem to have a [test](https://tests.caniuse.com/u2f), I didn't touch Edge & Opera for now
- general question: safari? didn't [find an issue](https://bugs.webkit.org/buglist.cgi?bug_status=__all__&content=u2f&no_redirect=1&order=Importance&query_format=specific)…

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Not sure about the validation failure…? 🤔

File: features-json/u2f.json [hunk 1]
```diff
@@ -264,15 +280,15 @@
...
-      "105":"y #3",
-      "106":"y #3"
+      "98":"d #3",
+      "99":"d #3",
+      "100":"d #3",
+      "101":"d #3",
+      "102":"d #3",
+      "103":"d #3",
+      "104":"d #3",
+      "105":"d #3",
```
Comment (Fyrd) [inline features-json/u2f.json:290]: These are missing an `n`, should be `n d #3` :)

Comment (Schweinepriester) [reply features-json/u2f.json]: whoops, one would think I should know that by now 🙃
thanks, fixed! :)

**Comment by Fyrd** [comment]
Looks good, thanks for the research!

**Comment by Fyrd** [comment]
@Schweinepriester Think we should change the status to `unoff`? The spec seems to have been a ["review draft"](https://fidoalliance.org/specs/fido-u2f-v1.0-nfc-bt-amendment-20150514/fido-u2f-javascript-api.html) so probably should already have been...

**Pull request merged**